src/sxastarter/src/components/play-travel/Experiences.tsx

```diff
@@ -29,17 +28,20 @@ export type ExperiencesProps = {
 
 export const Default = (props: ExperiencesProps): JSX.Element => {
   const id = props.params.RenderingIdentifier;
-  const { datasource, contextItem } = props.fields.data;
-  const data = datasource || contextItem;
-  const experiences = data ? (
+  const { datasource } = props.fields.data;
+  const experiences = datasource ? (
     <>
-      <div className="font-bold leading-tight text-2xl md:text-5xl pr-24 sm:pr-40 md:leading-tight">
-        <JssTextField field={data.title.jsonValue} />
+      <div className="font-bold leading-tight text-2xl md:text-5xl pr-24 sm:pr-0 xl:pr-40 md:leading-tight">
```

Comment:
Use pure function for this part. for example:
const ExperiencesRender = () => ();